Assets/PatchKit/UnityEditorExtension/Editor/UI/CreateAppScreen.cs

```diff
@@ -42,16 +42,50 @@ public override void UpdateIfActive()
 
     public override void Draw()
     {
-        GUILayout.Label(
-            string.Format(
-                "Create new PatchKit app for {0}",
-                _platform.ToDisplayString()),
-            EditorStyles.boldLabel);
-
-        _name = EditorGUILayout.TextField("Name: ", _name);
-
-        EditorGUILayout.Space();
+        if (GUILayout.Button(new GUIContent(_arrowIcon, "Change application"), GUILayout.Width(35), GUILayout.Height(20)))
+        {
+            Dispatch(() => Cancel());
+        }
 
+        EditorGUILayout.BeginHorizontal();
+        {
+            GUILayout.FlexibleSpace();
+            GUILayout.Label(
+                string.Format(
+                    "Create new PatchKit app for {0}",
+                    _platform.ToDisplayString()),
+                EditorStyles.boldLabel);
+            GUILayout.FlexibleSpace();
+        }
+        EditorGUILayout.EndHorizontal();
+        
+        
+        
+        GUILayout.BeginHorizontal();
+        {
+            GUILayout.Label("Name:");
+            _name = EditorGUILayout.TextField(_name);
+        }
+        GUILayout.EndHorizontal();
+        EditorGUILayout.BeginHorizontal();
+        {
+            GUILayout.Label(new GUIContent("Target platform:     " + EditorUserBuildSettings.activeBuildTarget,"PatchKit application target platform is unambiguous with current active project build platform."));
+            GUILayout.FlexibleSpace();
+            if (GUILayout.Button(
+                new GUIContent(
+                    "Switch Platform",
+                    "Change PatchKit application target platform."),
+                GUILayout.Width(110)))
+            {
+                if (EditorUtility.DisplayDialog("Warning", "To change application platform, you need to switch the project platform. \n\nRemember, PatchKit support only: Windows, Mac, Linux platforms.",

```

Comment:
Please split this string into multiple lines, it's hard to read it like that.

Reply:
ok